intro-Linux/ch_getting_started.tex

```diff
@@ -94,13 +93,23 @@ \section{Variables}
 \subsection{Defining variables}
 
 There are several variables already defined for you when you start your
-session, such as ``\$HOME'' which contains the path to your home directory. 
+session, such as ``\$HOME'' which contains the path to your home directory.
+
+For a full overview of defined environment variables in your current session,
+you can use the ``env'' command:
+
+\begin{prompt}
+%\shellcmd{env}%
```

Comment:
It's generally better to push this through `sort`, i.e. `env | sort` (unsorted output makes it hard to look for stuff), and often useful to `grep` for certain things like `env | sort | grep VSC`.

Maybe mention that too, i.e.

i) use `env | sort` here, and explain in one sentence why `sort` is used
ii) also mention (below) that you can `grep` for certain variable names or values, and give a short example (e.g. `env | sort | grep VSC` to find all VSC-specific environment variables

Try to keep it short though...